README.md

```diff
@@ -136,6 +136,34 @@ compile "com.android.support:appcompat-v7:$BAR"
 compile "com.android.support:palette-v7:$BAR"
 ```
 
+NOTE :
+The support libraries are now available through Google's Maven repository, so ensure that you have that added to your project's repositories.
```

Comment:
We should also link "Google's Maven repository" to here:
https://developer.android.com/studio/build/dependencies.html#google-maven